CMakeLists.txt

```
@@ -171,6 +180,25 @@ if(SDL2NET_BUILD_SHARED_LIBS)
     sdl_target_link_options_no_undefined(SDL2_net)
 endif()
 
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/SDL2_netConfigVersion.cmake"
+  VERSION ${FULL_VERSION}
+  COMPATIBILITY AnyNewerVersion
+)
+
+configure_package_config_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/SDL2_netConfig.cmake.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/SDL2_netConfig.cmake"
+  NO_SET_AND_CHECK_MACRO
+  INSTALL_DESTINATION "${CMAKE_CURRENT_BINARY_DIR}"
+  INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}"
+)
+
+export(TARGETS SDL2_net
+  NAMESPACE SDL2_net::
+  FILE "SDL2_net-${sdl2_net_install_name_infix}-targets.cmake"
+)
```

This can only be done when `SDL2::SDL2` is detected using `find_package`, and not when `SDL2_net` is added as a subproject alongside SDL2.
Let's only export the targets when the `SDL2` target is not available.
```suggestion
if(NOT SDL2)
  export(TARGETS SDL2_net
    NAMESPACE SDL2_net::
    FILE "SDL2_net-${sdl2_net_install_name_infix}-targets.cmake"
  )
endif()
```